docker/config_template.yaml

```diff
@@ -33,11 +33,15 @@ persistence:
                     serverName: {{ default .Env.CASSANDRA_HOST_NAME "" }}
         visibility:
             cassandra:
-                hosts: "{{ default .Env.CASSANDRA_SEEDS "" }}"
+                {{- $vis_seeds := default .Env.VISIBILITY_CASSANDRA_SEEDS .Env.CASSANDRA_SEEDS -}}
```

Comment:
I don't like abbreviations like this. Can you change `vis` to `visibility` everywhere.